evolver/history.py

```diff
@@ -18,10 +19,12 @@ def get(self, query):
 
 
 class HistoryServer(History):
-    def __init__(self, evolver=None, config=None):
-        self.evolver = evolver
-        self.config = config
-        self.history = defaultdict(list)
+    class Config(History.Config):
+        name: str = "HistoryServer"
```

Comment:
We could do, though, the issue with the default init mechanism is that there's an asymmetry between ``cls.Config()`` (where ``name=None``) and ``cls.create().config``, (where ``name=cls.__name__``).

So I added it here to allow for easier testing of ``test_empty_config_property_equivalence`` and since it's a out-of-the-box object, thought it ok to be explicit as example.